source/Weapon.h

```diff
@@ -57,6 +57,8 @@ class Weapon {
 
 
 public:
+	Weapon() = default;
+
 	// Load from a "weapon" node, either in an outfit, a ship (explosion), or a hazard.
 	void LoadWeapon(const DataNode &node);
 	bool IsWeapon() const;
```

Comment:
Do we need to have Weapon::IsWeapon? Presumably we only really need Outfit::IsWeapon.

Reply:
we need it for hazards too